tools/src/main/java/org/apache/kafka/tools/VerifiableProducer.java

```diff
@@ -189,6 +190,14 @@ private static ArgumentParser argParser() {
             .dest("repeatingKeys")
             .help("If specified, each produced record will have a key starting at 0 increment by 1 up to the number specified (exclusive), then the key is set to 0 again");
 
+        parser.addArgument("--command-config")
+            .action(store())
+            .required(false)
+            .type(String.class)
+            .metavar("CONFIG-FILE")
+            .dest("commandConfigFile")
+            .help("Config properties file.");
```

Comment:
And here `(config options shared with command line parameters will be overridden)`.